cypress/integration/customizedResources/user/add/AddRecord.spec.js

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker';
+import { customized, leftNavbar } from '../../../../support/cssCommonSelectors';
+import { common } from '../../../../support/texts';
+import { getFormValues } from '../../../../support/helpersMethods';
+import comp from '../../../../support/components';
+  
+const { inputs, boardView } = customized;
+let email = 'aaaaaa' + faker.internet.email().toLowerCase();
```

Comment:
```suggestion
const email = faker.internet.email();
```
why 'aaaa' on the beggining?

Some people might want to enter email in camel case, probably no need to lowercase that